src/borg/crypto.pyx

```diff
@@ -385,14 +385,17 @@ cdef class AES256_CTR_HMAC_SHA256:
 
     def set_iv(self, iv):
         # set_iv needs to be called before each encrypt() call
+        if isinstance(iv, int):
+            iv = iv.to_bytes(self.iv_len, byteorder='big')
```

Comment:
Do we still need iv as bytes? I wonder if we shouldn’t go the whole way and just have a python `int` as iv in the class and do the to_bytes only in store_iv and in calls to EVP_EncryptInit_ex/EVP_DecryptInit_ex.

Also it seems increment_iv is now unused